### PR TITLE
Adds setup script to backend Python connector package

### PR DESCRIPTION
**Pull request by cw75**
With this change, we can `cd` into `src/python` and install the latest development version of the backend Python package via `pip3 install .`.

### CODE REVIEW

File: src/python/setup.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import os
+
+from setuptools import find_packages, setup
+
+
+install_requires = open("requirements.txt").read().strip().split("\n")
+
+setup(
+    name="aqueduct-ml",
+    version="0.0.1",
```
Comment (saurav-c) [inline src/python/setup.py:11]: Is this something that should ever be changed, or is it supposed to be hard-coded?

**Review by saurav-c — APPROVED** [review]
LGTM! Left a question about versioning and just a reminder to add the `typing-extensions` package to the requirements.

File: src/python/setup.py (hunk 1 above)
Comment (cw75) [reply src/python/setup.py:11]: We should bump this when we are ready to cut another release.

**Comment by cw75** [comment]
@saurav-c added the package into the requirements file.

**Pull request merged**